### ISSUE TITLE
Create-React-App production build error: Uncaught ReferenceError: require is not defined

### ISSUE THREAD

**Issue by dmr07** [wojtekmaj/react-pdf#852]
**Before you start - checklist**

 - [x] I followed instructions in documentation written for my React-PDF version
 - [x] I have checked if this bug is not already reported
 - [x] I have checked if an issue is not listed in [Known issues](https://github.com/wojtekmaj/react-pdf/wiki/Known-issues)
 - [ ] If I have a problem with PDF rendering, I checked if my PDF renders properly in Mozilla Firefox

**Description**

Production build of Create-React-App logs the following error: 
```
Uncaught ReferenceError: require is not defined
```
Non-production build works just fine. 

**Steps to reproduce**

```js
import { Document, Page } from 'react-pdf/dist/umd/entry.webpack';
... 
useEffect(() => {
        console.log(activeNote); // this is valid
        console.log(activeNote.link); // this is also valid
}, [])

<Document file={{url: activeNote.link}}
                    loading={<div className="note-loading">Loading</div>}
                    onLoadSuccess={onDocumentLoadSuccess}>
```

**Expected behavior**

To display PDF

**Additional information**

If applicable, add screenshots (preferably with browser console open) and files you have an issue with to help explain your problem.

<img width="534" alt="screenshot 2021-09-18 at 6 25 32 PM" src="https://user-images.githubusercontent.com/6568322/133912534-a536f744-317c-4e6f-b8b8-df5def018274.png">

The line that the error references is this: 
```
var _possibleConstructorReturn = require("/Users/MyUser/Projects/myproject/node_modules/babel-preset-react-app/node_modules/@babel/runtime/helpers/possibleConstructorReturn")
  , _get = require("/Users/MyUser/Projects/myproject/node_modules/babel-preset-react-app/node_modules/@babel/runtime/helpers/get")
```

I find it odd that my local path is seen in the production. The build is done with:

`npm run build`

**Environment**

 - Browser (if applicable): Safari 14.1.2, Brave 1.29.79
 - React-PDF version: 5.3.2
 - React version: 17.0.2
 - Webpack version (if applicable):

**Comment by dmr07** [comment]
Adding the worker code fixed it. I thought this was optional. 
```js
import { pdfjs } from 'react-pdf';
pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.min.js`;
```

**Comment by TatianaMatumoto** [comment]
> Adding the worker code fixed it. I thought this was optional.
> 
> ```js
> import { pdfjs } from 'react-pdf';
> pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.min.js`;
> ```

that solved my problem, tranks dude!

**Comment by tarikt-hybird** [comment]
> Adding the worker code fixed it. I thought this was optional.
> 
> ```js
> import { pdfjs } from 'react-pdf';
> pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.min.js`;
> ```

thank you @dmr07 ! Your solution worked for me. I was able to reproduce this issue only on production environments so I ran it locally with `yarn build` and `serve -s build` to mimic the production build. This gave me the `require is not defined error`. After applying your suggestion, the issue has been resolved both locally and on production env. Thanks!

My setup: 

"react": "^16.8.6",
"react-pdf": "^7.4.0"
